src/sentry/api/endpoints/artifact_lookup.py

```diff
@@ -133,8 +134,10 @@ def update_bundles(inner_bundles: Set[Tuple[int, datetime, int]]):
                 bundle_file_ids |= get_legacy_release_bundles(release, dist)
                 individual_files = get_legacy_releasefile_by_file_url(release, dist, url)
 
-        # Before constructing the response, we want to asynchronously update the artifact bundles renewal date.
-        renew_artifact_bundles(used_artifact_bundles)
+        if options.get("sourcemaps.artifact-bundles.asynchronous-renewal") == 1.0:
```

Comment:
if you want to roll this out gradually, you can also sample this via `random`.

Reply:
I don't think we want to sample the rollout, do you think it would make sense?